README.md

```diff
@@ -34,42 +34,50 @@ This fork aims to be community managed, meaning that the decisions are taken
 in a collegial fashion, and that a bunch of maintainers should be able to make
 the review process quicker and more streamlined.
 
-## Install
+## Installation and usage
 
-To use The Lounge you must have [Node.js](https://nodejs.org/en/download/) installed.
-The oldest Node.js version we support is 4.2.0.
+The Lounge requires [Node.js](https://nodejs.org/) v4 or more recent.
 
-If you still use 0.10 or 0.12 we strongly advise you to upgrade before installing The Lounge.
-For more information on how to upgrade, read the [documentation](https://nodejs.org/en/download/package-manager/).
+### Running stable releases from npm (recommended)
 
-```
+Run this in a terminal to install (or upgrade) the latest stable release from
+[npm](https://www.npmjs.com/):
+
+```sh
 sudo npm install -g thelounge
 ```
 
-## Usage
-
-When the install is complete, go ahead and run this in your terminal:
+When install is complete, run:
```

Comment:
I think this should stay as "When the install", alternatively you could use "When installation is..."